### PR TITLE
Discrete panel

### PR DESCRIPTION
**Pull request by milkpirate**
## What does this do?
Adds basic support for Discrete https://grafana.com/grafana/plugins/natel-discrete-panel/ including panel itself and color mappings (DiscreteColorMappingItem to keep things tidy).

## Why is it a good idea?
Adds "Discrete".

## Context
Discrete inherits from Panel to keep the changes small.

## Questions
- How extensiv are you testing?
- Do these tests suffice?
- Any suggestions to use pre-existing features instead of DiscreteColorMappingItem?

### CODE REVIEW

File: grafanalib/core.py [hunk 1]
```diff
@@ -1452,9 +1453,57 @@ def auto_ref_ids(self):
...
+        ),
+    )
+
+    def to_json_data(self):
+        graphObject = {
+            'colorMaps': self.colorMapsItems,
+            'datasource': self.dataSource,
+            'grid': self.gridPos,
+            'targets': self.targets,
+            'title': self.title,
```
Comment (JamesGibo) [inline grafanalib/core.py:1501]: These lines can all be removed as they will be inherited from the Panel class
```suggestion
```

**Review by JamesGibo — CHANGES_REQUESTED** [review]
Thanks for this PR, just one minor change to remove code not required

File: grafanalib/core.py (hunk 1 above)
Comment (milkpirate) [reply grafanalib/core.py]: Accepted.

**Review by JamesGibo — APPROVED** [review]

**Comment by JamesGibo** [comment]
Thanks for your contribution @milkpirate

**Comment by milkpirate** [comment]
I might add functionality in the future for the many switches the panel supports:
image

**Pull request merged**